frontend/mr-admin-flate/src/components/tiltaksgjennomforinger/opprett/OpprettTiltaksgjennomforingContainer.tsx

```diff
@@ -0,0 +1,244 @@
+import { zodResolver } from "@hookform/resolvers/zod";
+import { formaterDatoSomYYYYMMDD } from "../../../utils/Utils";
+import { Datovelger } from "../../skjema/OpprettComponents";
+import styles from "./OpprettTiltaksgjennomforingContainer.module.scss";
+import { FormGroup } from "../../avtaler/opprett/OpprettAvtaleContainer";
+import { Avtale, NavEnhet, Tiltakstypestatus } from "mulighetsrommet-api-client";
+import { mulighetsrommetClient } from "../../../api/clients";
+import { useAtom } from "jotai";
+import { avtaleFilter, tiltakstypefilter } from "../../../api/atoms";
+import { TiltaksgjennomforingRequest } from "mulighetsrommet-api-client/build/models/TiltaksgjennomforingRequest";
+import { useAvtaler } from "../../../api/avtaler/useAvtaler";
+import { useAlleEnheter } from "../../../api/enhet/useAlleEnheter";
+import useTiltakstyperWithFilter from "../../../api/tiltakstyper/useTiltakstyperWithFilter";
+
+const Schema = z.object({
+  tiltakstype: z.string().min(1, "Du må velge en tiltakstype"),
+  avtale: z.string().min(1, "Du må velge en avtale"),
+  tittel: z.string().min(1, "Du må skrive inn tittel"),
+  tiltaksnummer: z
+    .number({ invalid_type_error: "Du må skrive inn et tall" })
+    .int(),
+  aar: z
+    .number({ invalid_type_error: "Du må skrive inn et tall" })
+    .int(),
+  fraDato: z.date({ required_error: "En gjennomføring må ha en startdato" }),
+  tilDato: z.date({ required_error: "En gjennomføring må ha en sluttdato" }),
+  antallPlasser: z
+    .number({
+      invalid_type_error:
+        "Du må skrive inn antall plasser for gjennomføringen som et tall",
+    })
+    .int(),
+  enhet: z.string().min(1, "Du må velge en enhet"),
+  tiltakssted: z.string({ required_error: "Du må skrive inn tiltakssted for gjennomføringen" }),
+});
+
+export type inferredSchema = z.infer<typeof Schema>;
+
+interface OpprettTiltaksgjennomforingContainerProps {
+  setError: Dispatch<SetStateAction<string | null>>;
+  setResult: Dispatch<SetStateAction<string | null>>;
+}
+
+export const OpprettTiltaksgjennomforingContainer = (
+  props: OpprettTiltaksgjennomforingContainerProps
+) => {
+  const form = useForm<inferredSchema>({
+    resolver: zodResolver(Schema),
+  });
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = form;
+
+  const [aFilter, setAvtaleFilter] = useAtom(avtaleFilter);
+  const [tFilter, setTiltakstypeFilter] = useAtom(tiltakstypefilter);
+  useEffect(() => {
+    setTiltakstypeFilter({...tFilter, status: Tiltakstypestatus.AKTIV})
+  })
+
+  const { data: tiltakstyper, isLoading: isLoadingTiltakstyper, isError: isErrorTiltakstyper } = useTiltakstyperWithFilter();
+
+  const { data: enheter, isLoading: isLoadingEnheter, isError: isErrorEnheter } = useAlleEnheter();
+  const { data: avtaler, isLoading: isLoadingAvtaler, isError: isErrorAvtaler } = useAvtaler();
+  if (isErrorAvtaler || isErrorEnheter || isErrorTiltakstyper) {
+    props.setError("error");
```

Comment:
Klarer vi en bedre feilmelding enn "error" her? Hva med `Det oppsto en feil ved henting av nødvendig data for opprettelse av tiltaksgjennomføring`?

Reply:
Den stringen brukes ikke, det vises 
```
Gjennomføringern kunne ikke opprettes på grunn av en teknisk feil hos oss.
                Forsøk på nytt eller ta <a href={porten}>kontakt</a> i Porten
                dersom du trenger mer hjelp.
```
på alle feil. Kan endre den til å være boolean i stedet, det er litt mindre misvisende. Evt. gjøre den feilmeldingen faktisk litt mer fleksibel

Reply:
Ah, skjønner. Ja, kanskje greit med boolean da. Ble fintet ut siden du noen andre steder har en mer forklarende feilmelding (som heller ikke blir brukt).

Reply:
Ja sorry det var veldig misvisende 😅